src/%IPM/Utils/Module.cls

```diff
@@ -270,7 +271,10 @@ ClassMethod LoadModuleReference(pServerName As %String, pModuleName As %String,
 				If $$$ISERR(tSC) {
 					Quit
 				}
-				
+        If $Get(pParams("Install")),pModuleName'="zpm",tInstModule.DeveloperMode,'$Get(pParams("DeveloperMode"),0) {
```

Comment:
This hard-coded reference to zpm module name needs to use parameter for ipm module name (forgot where that is)